.releaserc

```diff
@@ -0,0 +1,39 @@
+{
+  "branches": [
+    "master",
+    {
+      "name": "feat/cc-widgets",
+      "prerelease": "wxcc"
+    }
+  ],
+  "plugins": [
+    "@semantic-release/commit-analyzer",
+    "@semantic-release/release-notes-generator",
+    [
+      "@semantic-release/changelog",
+      {
+        "changelogFile": "packages/contact-center/CHANGELOG.md"
+      }
+    ],
+    [
+      "@semantic-release/exec",
+      {
+        "prepareCmd": "node ./tooling/src/publish.js ${nextRelease.channel} ${nextRelease.version} "
+      }
+    ],
```

Comment:
Leaving this commit to highlight that this part needs to be added once we have added 2 releases onto github.